src/org/labkey/test/components/react/dialogs/SampleDeleteDialog.java

```diff
@@ -0,0 +1,18 @@
+package org.labkey.test.components.react.dialogs;
+
+import org.labkey.test.components.experiment.DeleteConfirmationDialog;
+import org.labkey.test.pages.LabKeyPage;
+import org.labkey.test.pages.samplemanagement.samples.SampleTypePage;
+
+public class SampleDeleteDialog<SourcePage extends LabKeyPage> extends DeleteConfirmationDialog<SourcePage, SampleTypePage>
```

Comment:
The build is failing because this references code in `samplemanagment`. `DeleteConfirmationDialog` is the shared component. This is the sample manager specific implementation.

Reply:
Ha, mildly infuriating to learn that there's a shared component for this after all.

Reply:
Just sent it back to SM, refactored SM tests to point back to the old location.
Hopefully this doesn't break Biologics/require me to implement a similar pattern over there